python-sdk/tests/benchmark/Makefile

```diff
@@ -5,7 +5,7 @@ GCP_PROJECT ?= astronomer-dag-authoring
 GIT_HASH = $(shell git log -1 --format=%h)
 APP ?= benchmark
 CONTAINER_REGISTRY=gcr.io/$(GCP_PROJECT)/$(APP)
-
+PLATFORM="linux/amd64"
```

Comment:
This is not configurable right now, or? Everyone using this script has to run it on m1. Maybe we should make it configurable through an env var.

Reply:
yes, it is not configurable. I think the platform option in the docker build is more about the target machine where we want to run this image?
I think it's ok for now but let me know if you want this value from env I'm happy to do that change and keep `linux/amd64` as the default value

Reply:
Yes, I think some of us do not have a m1.

Reply:
So it would be nice to be able to set it without editing the script.

Reply:
PR should take care of this https://github.com/astronomer/astro-sdk/pull/1002